diff --git a/node_modules/@cypress/code-coverage/support.js b/node_modules/@cypress/code-coverage/support.js
index 9aaa014..4a87e8b 100644
--- a/node_modules/@cypress/code-coverage/support.js
+++ b/node_modules/@cypress/code-coverage/support.js
@@ -60,7 +60,7 @@ const registerHooks = () => {
     })
   })
 
-  beforeEach(() => {
+  afterEach(() => {
     // each object will have the coverage and url pathname
     // to let the user know the coverage has been collected
     windowCoverageObjects = []
